### PR TITLE
adding in preprod and prod rules from planet FM

### PR DESCRIPTION
**Pull request by haitchison**
## A reference to the issue / Description of it

Including a FW rule that allows traffic from DOM1 domain controllers (10.172.68.146/29) on port 9389 to private subnets, specifically for Planet FM preprod and prod. 

## How does this PR fix the problem?

New Preprod and Prod firewall rule, with domain-controller cidr range

## How has this been tested?

Firewall logs indicate that this traffic is currently blocked, will re-test by running connection tests from prod / preprod planet servers.

## Deployment Plan / Instructions

It will add an additional allow rule to the MP firewall

## Checklist (check `x` in `[ ]` of list items)

- [x] I have performed a self-review of my own code
- [x] All checks have passed
- [ ] I have made corresponding changes to the documentation
- [x] Plan and discussed how it should be deployed to PROD (If needed)

## Additional comments (if any)

{Please write here}

### CODE REVIEW

**Review by markgov — APPROVED** [review]
LGTM

File: terraform/environments/core-network-services/firewall-rules/preproduction_rules.json
Comment (dms1981) [inline terraform/environments/core-network-services/firewall-rules/preproduction_rules.json:1]: Hi @haitchison! I think you're actually after this:
```
    "source_ip": "${hmpps-preproduction}",
    "destination_ip": "${dom1-domain-controllers}",
```
As I understand the flow, the source will be your instances in the `hmpps-preproduction` VPC. Also the local that generates the platform CIDR ranges produces them as `$business_unit-$environment` so I've trimmed the VPC name down a bit in my example.

File: terraform/environments/core-network-services/firewall-rules/production_rules.json
Comment (dms1981) [inline terraform/environments/core-network-services/firewall-rules/production_rules.json:1]: As above - I think you're actually after this:
```
    "source_ip": "${hmpps-preproduction}",
    "destination_ip": "${dom1-domain-controllers}",
```

Comment (haitchison) [reply terraform/environments/core-network-services/firewall-rules/production_rules.json:1]: Updated! :)

**Review by dms1981 — APPROVED** [review]

**Pull request merged**